Jenkinsfile

```diff
@@ -48,6 +48,8 @@ devToolsProject.run(
     data['dtrImage'].deploy(
       '8000',
       '-v jenkins-nodes:/jenkins_nodes',
+      '--target-port 9100',
+      '--target-port 9323',
```

Comment:
`deploy` takes a maximum of 3 arguments, here we are passing 5.

Reply:
Fixed in de1ea21.